devtools/helpers/smartrequests.py

```diff
@@ -284,6 +284,15 @@ def get_preset_rules(params: GetRulesParams | None = None) -> SmartRequest:
         """Get preset rules."""
         return SmartRequest("get_preset_rules", params or SmartRequest.GetRulesParams())
 
+    @staticmethod
+    def get_on_off_gradually_info(
+        params: SmartRequestParams | None = None,
+    ) -> SmartRequest:
+        """Get preset rules."""
+        return SmartRequest(
+            "get_on_off_gradually_info", params or SmartRequest.SmartRequestParams()
+        )
```

Comment:
This still needs to be tested as working with the affected devices. Will send an empty `{}` instead of `null` for params.

Reply:
`get_on_off_gradually_info` now confirmed as working in https://github.com/python-kasa/python-kasa/issues/1033